### DISCUSSION TITLE
Javadoc

### DISCUSSION THREAD

**Discussion by matthew16550** [plantuml/plantuml#813] [General]
This is a collecting point for javadoc discussions

**Comment by matthew16550** [comment]
I wonder if we should exclude the vendored dependencies from javadoc.jar.

There are many javadoc warnings from `src/jcckit/*` and `/src/ext/plantuml/com/google/zxing/*` and because they are vendored we probably don't want to change them too much.

Perhaps we should only make javadoc from `src/net/sourceforge/plantuml/*` ?

**Reply by arnaudroques** [reply]
> Perhaps we should only make javadoc from `src/net/sourceforge/plantuml/*` ?

Yes, we must.
Let's exclude the other dependencies!

**Comment by matthew16550** [comment]
Hi @arnaudroques

What is the situation with `maven-javadoc-plugin`,  I see in #695  and c8748421584d3c0edb734c271721ed0a0b8e79a0 that we are staying with `3.2.0` ?

**Reply by arnaudroques** [reply]
It's because I had issues with the vendored dependencies.
If we remove them, we could probably go ahead
